### ISSUE TITLE
can't mod probe zfs on 2020 release

### ISSUE THREAD

**Issue by ghost** [danboid/ALEZ#53]
Just fetched archlinux-alez-2020.02.29.03.21.57 release to try in a vm and was unable to modprobe zfs 

![2020-02-29_00-22](https://user-images.githubusercontent.com/980/75601488-a2688680-5a89-11ea-8ed1-778a3565ce29.png)

**Comment by johnramsden** [comment]
Rebuild in progress, will update this issue as soon as it's available. It's an issue that was caused by switching to be `zfs-linux-lts` kernel and not installing the group `archzfs-linux-lts`.

**Comment by johnramsden** [comment]
Originally I thought I would just switch to the LTS kernel but that proved to be much more difficult since the kernel is actually hard-coded within the archiso build scripts.

Currently ZFS is facing https://github.com/archzfs/archzfs/issues/328 which means updating the stable kernel is not working, temporarily I'm setting up a build with the oldest working stable kernel.

**Comment by ghost** [comment]
Thanks, I'll give this a go once the build is complete.

**Comment by johnramsden** [comment]
Released [archlinux-alez-2020.02.29.08.24.04](https://github.com/danboid/ALEZ/releases/tag/archlinux-alez-2020.02.29.08.24.04). Let me know if it fixes the issue for you

**Comment by ghost** [comment]
I'm able to modprobe zfs now and the installer is downloading packages.

**Comment by ghost** [comment]
Take that back, soon as I hit enter to install the packages and flipped over here to tell you it was ok it blew up.

![2020-02-29_03-53](https://user-images.githubusercontent.com/980/75604489-fd719c00-5ad0-11ea-84df-48be13acb3ce.png)

**Comment by johnramsden** [comment]
It might be due to the kernel problem I mentioned, does the LTS version work?

**Comment by ghost** [comment]
Same problem with LTS.

**Comment by johnramsden** [comment]
Will do some testing and see if I can reproduce.

**Comment by johnramsden** [comment]
An error was introduced as a side effect of using a custom local repository to install the older version of the kernel, which caused an error during mirror sync. Upstream just released the new up-to-date stable kernel https://github.com/archzfs/archzfs/issues/328#issuecomment-592954068, so I removed the work around and installer appears to be working - tested full install.

Let me know if the [new release](https://github.com/danboid/ALEZ/releases/tag/archlinux-alez-2020.02.29.20.02.48) works for you.

**Comment by ghost** [comment]
Worked great that time. Now quick question, am I going to run into breakage every time they update stable? What's the best way to avoid all this once it's installed on my desktop? I'm a little confused by some of it.

**Comment by johnramsden** [comment]
No, you won't. I was trying to get a quick fix in and should have done more extensive testing before releasing the ISO - mybad. Most of the problems above were unrelated to archzfs.

You _will_ occasionally get `zfs-linux` _slightly_ delayed behind `linux` releases - I've found it's usually just a few hours, occasionally longer. The problem linked above was due to an upstream openzfs bug https://github.com/openzfs/zfs/issues/9745 due to kernel changes that caused problems during builds of ZFS that meant archzfs had to wait to release a new kernel. This is a very occasional problem, and if you are concerned with getting out of sync, using the LTS kernel would be an option since it's update cadence is quite lower. A good option is to stay on the stable stream and if an upstream bug occurs that causes builds to be delayed switch to the LTS kernel.
